kpconv_torch/io/ply.py

```diff
@@ -23,18 +24,8 @@
     b"double": "f8",
 }
 
-# Numpy reader format
-valid_formats = {"ascii": "", "binary_big_endian": ">", "binary_little_endian": "<"}
-
 
-# ----------------------------------------------------------------------------------------------------------------------
-#
-#           Functions
-#       \***************/
-#
-
-
-def parse_header(plyfile, ext):
+def parse_ply_header(plyfile, ext):
```

Comment:
Side note regarding the function name modification: handling `.ply` files there is implicit, as we are in the `ply.py` module.